### ISSUE TITLE
Add Accessibility icon for relevant properties 

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#819]
We recently removed the 'Special Needs' icon that we had on select properties (see #762), as some users found the term offensive. In lieu of that term, we should add an Accessible icon that makes it clear that the property has features catering to users with disabilities. The logic for when it displays should be the same as the previous 'Special Needs' icon.

Design is available in [Figma](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497) (01.1).
![accessible_icon](https://user-images.githubusercontent.com/394908/145121286-eb8f3282-9284-4bc4-afba-d7753302e24e.png)
